tests/Unit/Support/Constraint/CssConstraintTest.php

```diff
@@ -129,4 +130,32 @@ public function getCssNeedleRegularExpressionPatternInsertsOptionalWhitespaceAft
 
         self::assertSame('/\\<style\\>\\s*+a/', $result);
     }
+
+    /**
+     * @return string[][]
```

Comment:
We can make this even more specific:
`array<string, array<string>>`

Reply:
Done.